gitpod-io/gitpod
# Add real owner to imagebuild metadata

## Description
Add real owner to imagebuild metadata

## Related Issue(s)
n.a.

## How to test
- Start image build in preview environment e.g. https://fo-imgbuild-owner.staging.gitpod-dev.com/#imagebuild/https://github.com/gitpod-io/gitpod
- Verify that owner label on the pod is the id of the user that triggered the imagebuild and not image-builder
```
Labels:       app=gitpod
              component=workspace
              gitpod.io/networkpolicy=default
              gpwsman=true
              headless=true
              metaID=db0de155-4a44-4d24-874a-f28d219dfeb1
              owner=b8a478cf-8488-44e1-a735-c155086c5095
              workspaceID=db0de155-4a44-4d24-874a-f28d219dfeb1
              workspaceType=imagebuild
```
- Use gcptl imagebuilds list -> imagebuild should still be listable

## Notes for reviews
The push of the imagebuild to the registry fails in the preview environment, but this also happens without my changes.

## Release Notes

```release-note
None
```
